config/configtls/configtls.go

```diff
@@ -175,16 +180,39 @@ func (c TLSSetting) loadTLSConfig() (*tls.Config, error) {
 	if err != nil {
 		return nil, fmt.Errorf("invalid TLS max_version: %w", err)
 	}
+	cipherSuites, err := convertCipherSuites(c.CipherSuites)
+	if err != nil {
+		return nil, err
+	}
 
 	return &tls.Config{
 		RootCAs:              certPool,
 		GetCertificate:       getCertificate,
 		GetClientCertificate: getClientCertificate,
 		MinVersion:           minTLS,
 		MaxVersion:           maxTLS,
+		CipherSuites:         cipherSuites,
 	}, nil
 }
 
+func convertCipherSuites(cipherSuites []string) ([]uint16, error) {
+	var result []uint16
+	for _, suite := range cipherSuites {
+		found := false
+		for _, supported := range tls.CipherSuites() {
+			if suite == supported.Name {
+				result = append(result, supported.ID)
+				found = true
+				break
+			}
+		}
+		if !found {
+			return nil, fmt.Errorf("invalid TLS cipher suite: %q", suite)
```

Comment:
could this join errors instead of failing on the first one? this would be helpful in the case where there are multiple invalid cipher suites specified

Reply:
Sounds good, added code towards supporting this.